integration/raft/config_test.go

```diff
@@ -124,8 +125,11 @@ var _ = Describe("EndToEnd reconfiguration and onboarding", func() {
 				Eventually(sess, network.EventuallyTimeout).Should(gexec.Exit(0))
 			}
 
-			By("Creating a new channel")
-			network.CreateChannel("testchannel", orderer1, peer)
+			By("Creating a new channel, joining all orderers")
+			for _, o := range network.Orderers {
+				channelparticipation.JoinOrdererAppChannelCluster(network, "testchannel", o)
```

Comment:
Everytime the call to JoinOrdererAppChannelCluster loads block from the disk & unmarshals the blocks, if we move out this step outside this function & make function to cache or return the block, we can avoid the overhead of reading the block more than one in same iteration.

Reply:
Done. I pulled the loop into the function such that the block is loaded only once.